### DISCUSSION TITLE
FOSS alternatives

### DISCUSSION THREAD

**Discussion by NamelessGO** [FossifyOrg/General-Discussion#10] [Ideas]
Simple tools alternative 

https://you-apps.net/

https://github.com/you-apps

https://www.reddit.com/r/SimpleMobileTools/comments/18929pq/simplemobiletools_was_sold_alternatives/?rdt=62751

-----

Copy and paste the lines below and comment your project to replace

Calculator ->
Voice recorder ->
Flashlight ->
Music player ->
Keyboard -> 
Gallery ->
SMS ->
Calendar ->
Launcher ->
File manager ->
Clock ->
Notes ->
Draw ->
Camera ->
Contacts ->
App launcher ->

**Comment by NamelessGO** [comment]
Calculator ->
Voice recorder ->
Flashlight ->
Music player ->
Keyboard -> 
Gallery -> https://github.com/deckerst/aves
SMS ->
Calendar -> https://github.com/Etar-Group/Etar-Calendar
Launcher -> https://github.com/MM2-0/Kvaesitso
File manager -> https://github.com/zhanghai/MaterialFiles
Clock ->
Notes -> https://github.com/gsantner/markor
Draw ->
Camera ->
Contacts ->
App launcher -> https://github.com/MM2-0/Kvaesitso

**Comment by inson1** [comment]
https://www.reddit.com/r/SimpleMobileTools/comments/18929pq/simplemobiletools_was_sold_alternatives/

**Comment by NamelessGO** [comment]
Simple tools alternative 

https://you-apps.net/

https://github.com/you-apps

**Comment by inson1** [comment]
@NameLessGO Hey maybe this discussion would be better on reddit/lemmy/SMT github? I dont see how it adds to this project. :)
I dont mean to offend you in any way. I just think maybe this has its own place elsewhere? :)

**Comment by NamelessGO** [comment]
@inson1

Sorry, my intentions are to reduce the amount of apps we are forking like was said previously, the fork will happens to a few but important apps, so for the others it is best to look for an alternative already available

I don't use reddit, but happy to see someone had the same idea there

Planning to sharing on Lemmy once we get the first release on f-droid

**Comment by inson1** [comment]
@NameLessGO Ok in that case it makes sense :) But I would add that info to title or description :) My reddit post is maybe good and I'm going to try make it better :)

I think best Android Calculator is OpenCalc. And SMT calculator isnt good, so I wouldnt continue on that project.

**Comment by NamelessGO** [comment]
@inson1 It finally hit me that you wrote the post

It was a good post, thanks for bringing attention to this fork

Keep writing more post for Open Source projects in the reddit community, keeping them engaged in FOSS <3

**Comment by inson1** [comment]
@NameLessGO 
You didnt have to close it. It had purpose. You could help find out the best alternative and say if it is better or worse than SMT.

Feel free to repost (not the reddit post, but the text) (but please try update it if I add more alternatives)
And thank you :)
